.github/workflows/triage-unlabelled.yml

```diff
@@ -19,16 +19,25 @@ jobs:
     steps:
       - name: Check if issue is already in "${{ env.BOARD_NAME }}"
         run: |
-          if curl -i -H 'Content-Type: application/json' -H "Authorization: bearer ${{ secrets.GITHUB_TOKEN }}" -X POST -d '{"query": "query($issue: Int!, $owner: String!, $repo: String!) { repository(owner: $owner, name: $repo) { issue(number: $issue) { projectCards { nodes { project { name } } } } } } ", "variables" : "{ \"issue\": '${ISSUE}', \"owner\": \"'${OWNER}'\", \"repo\": \"'${REPO}'\" }" }' https://api.github.com/graphql | grep "\b$BOARD_NAME\b"; then
-            echo "Issue is already in Project '$BOARD_NAME', proceeding";
-            echo "ALREADY_IN_BOARD=true" >> $GITHUB_ENV
-          else
-            echo "Issue is not in project '$BOARD_NAME', cancelling this workflow"
-            echo "ALREADY_IN_BOARD=false" >> $GITHUB_ENV
+          json=$(curl -s -H 'Content-Type: application/json' -H "Authorization: bearer ${{ secrets.GITHUB_TOKEN }}" -X POST -d '{"query": "query($issue: Int!, $owner: String!, $repo: String!) { repository(owner: $owner, name: $repo) { issue(number: $issue) { projectCards { nodes { project { name } isArchived } } } } } ", "variables" : "{ \"issue\": '${ISSUE}', \"owner\": \"'${OWNER}'\", \"repo\": \"'${REPO}'\" }" }' https://api.github.com/graphql)
+          echo $json
```

Comment:
Do we still need this debug-looking thing?

Reply:
probably not. Will remove.